scripts/clear-all-fault-leds.sh

```diff
@@ -4,6 +4,14 @@
 # /xyz/openbmc_project/State/Decorator/OperationalStatusManager
 # to true or false.
 
+# Skip running this script during a warm reboot(graceful/force).
+current_chassis_status=$(busctl get-property xyz.openbmc_project.State.Chassis /xyz/openbmc_project/state/chassis0 xyz.openbmc_project.State.Chassis CurrentPowerState | cut -d" " -f2)
```

Comment:
Have you tested a reboot of the BMC when the host is up? Are we sure that the chassis0 object is on D-Bus when this script runs? Do we need to add a mapper-wait?

Reply:
The obmc-clear-all-faults service itself isn't executed during a reset-reload.
From the journal I could see only the slice unit is getting removed and created during reset-reload.

May 09 10:59:16 perfrain86bmctest systemd[1]: Removed slice Slice /system/obmc-clear-all-fault-leds-and-remove-crit-association.
May 09 10:59:59 perfrain86bmctest systemd[1]: Created slice Slice /system/obmc-clear-all-fault-leds-and-remove-crit-association.

Reply:
Verified that the service itself is not executing completely due to ConditionPathExists condition in the service file.
So we are good here.